src/shillelagh/backends/apsw/vt.py

```diff
@@ -317,8 +352,14 @@ def BestIndex(  # pylint: disable=too-many-locals
                     indexes.append((column_index, sqlite_index_constraint))
                     break
             else:
-                # no indexes supported in this column
-                constraints_used.append(None)
+                if (operator == Operator.LIMIT and self.adapter.supports_limit) or (
+                    operator == Operator.OFFSET and self.adapter.supports_offset
+                ):
+                    constraints_used.append((filter_index, True))
```

Comment:
Does setting this to true / false matter? Are there cases where it should be false? Does it differ for limit vs offset?

Reply:
It matters, but I couldn't find documentation about it. I had to set it to true for it work correctly.

Reply:
I found these docs (at least for offset): https://www.sqlite.org/vtab.html#omit_constraint_checking_in_bytecode